branding/css/susemanager-theme.less

```diff
@@ -143,13 +143,15 @@ section {
     position: fixed;
     #nav nav ul.level1 {
       overflow-y: auto !important;
+      scrollbar-color: rgba(173,216,230,0.5) transparent;
+      scrollbar-width: thin;
       &::-webkit-scrollbar {
         width: 7px;
       }
       &::-webkit-scrollbar-thumb {
         border-radius: 10px;
-        -webkit-box-shadow: inset 0 0 6px rgba(0,178,226,0.5);
-        background: rgba(0,178,226,0.5);
+        -webkit-box-shadow: inset 0 0 6px rgba(173,216,230,0.5);
```

Comment:
Why did you changed colors? Even if I like a little bit more the new one, :slightly_frowning_face:  I'd better keep the old one in order to be consistent and inline with the [SCC](http://scc.suse.com/) one

Reply:
Just because it looked prettier. It was looking bad in Firefox. The scc one looks ugly :P

I can revert it to the previous color :)